shared/src/native-lib/coreclr/src/pal/inc/rt/specstrings.h

```diff
@@ -314,7 +314,7 @@ __ANNOTATION(SAL_failureDefault(enum __SAL_failureKind));
 #define __deallocate(kind)                  _Pre_ __notnull __post_invalid
 #define __deallocate_opt(kind)              _Pre_ __maybenull __post_invalid
 #endif
-#define __bound                             __inner_bound
+//#define __bound                             __inner_bound
```

Comment:
Maybe remove the comment if not needed anymore?

Reply:
Since it's CLR headers, I prefer to keep it. When we update we will have to do the same thing.
But I least, I forgot to update the doc to explain why it's commented.

Reply:
Oh the readme was already updated.